src/components/Links/Links.tsx

```diff
@@ -12,7 +12,7 @@ export function Links(props: LinksProps) {
                 href={link.linkUrl} 
                 target="_blank"
                 colorSet={link.colorSet!}
-                key={link.linkUrl}>
+                key={index}>
```

Comment:
Thanks, this mistake was spotted. otherwise it would have caused big errors and bugs inside the code in the future 😃